setup.cfg

```diff
@@ -3,4 +3,4 @@ test = pytest
 
 [tool:pytest]
 addopts = -v -rf --disable-warnings
-python_files = tests/*.py
+python_files = tests/*/test_*.py
```

Comment:
I see that this can be a list (https://docs.pytest.org/en/stable/reference.html#confval-python_files), e.g.
`python_files = tests/*.py tests/*/test_*.py`
are you sure the new path include everything that the old path included ? why this change is needed

Reply:
Because this imported the assets in to pytest, and so the training files that import `tensorflow` an `horovod` failed cause the packages aren't installed.
And yes I checked, we didn't loose coverage. Everything is by convention with the `test_` prefix.